docs/source/Lines_functionality.ipynb

```diff
@@ -0,0 +1,1861 @@
+{
```

Comment:
<p>Given that the ek500 colormap made the bottom very red, and the red points became invisible: it is better to switch to anotehr color for the points.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OSOceanAcoustics/echoregions/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>